### ISSUE TITLE
Render overlapping bus lines

### ISSUE THREAD

**Issue by amenk** [clemosm/JustGo#2]
When there are overlapping bus lines in different colors, only the topmost is visible.

No clue how this could be solved ... maybe using a striped display of the different colors?

![image](https://user-images.githubusercontent.com/1087128/78473369-5a9be580-7740-11ea-87f3-878559afd011.png)

**Comment by clemosm** [comment]
Yes I know this issue... And I don't think there is a good way to do this in mapbox (beacause you would need to get the list of the bus lines for each way each time to set an offset for the line, or different colors). It is possible to display buses in dashed lines, but it's not very readable.
What I did is distinguish between "express" (passenger=regional) buses and others, to display them with a different offset. I could also display the main lines thicker and behind normal lines, but I have no good way to know which are the main ones for the moment.

**Comment by amenk** [comment]
Offsets would cool but I realize it's pretty hard to do https://medium.com/@mondaymaps/creating-a-route-map-with-offsets-876834a13b0e

**Comment by clemosm** [comment]
Good tutorial here, but yes it's quite a lot of work. I could work even with mapbox for example with modifying the data itself after it is downloaded, but I don't have enough time to work on that... I'll just add it to the "to-do" list for the moment
